src/jreepad/editor/EditPopupHandler.java

```diff
@@ -17,21 +17,23 @@
 /**
  *
  * @author not attributable
- * @version 1.0
+ * @version 1.1
  */
 public class EditPopupHandler
         extends MouseAdapter {
     private JPopupMenu popupMenu = new JPopupMenu();
     public static final String ITEM_CUT = "cut";
     public static final String ITEM_COPY = "copy";
     public static final String ITEM_PASTE = "paste";
+	public static final String ITEM_SELECTALL = "selectall";
```

Comment:
Here and elsewhere I see whitespace errors (tabs vs spaces). Could you please ensure your whitespace usage matches the rest of the file?